ion-schema/src/system.rs

```diff
@@ -2167,6 +2191,90 @@ mod schema_system_tests {
         assert!(&schema.is_err());
     }
 
+    #[test]
+    fn new_schema_test() {
+        let mut schema_system = SchemaSystem::new(vec![]);
+        let schema = schema_system.new_schema(
+            r#"
+                $ion_schema_2_0
+                schema_header::{}
+                
+                type::{
+                  name: my_type,
+                  type: string,
+                }
+                
+                schema_footer::{}
+            "#
+            .as_bytes(),
```

Comment:
You can use the [`b` string literal prefix](https://doc.rust-lang.org/reference/tokens.html#byte-string-literals) to get a `&[u8]` from a `&str`:
```rust
fn takes_bytes(_data: &[u8]) {}

fn main() {
    takes_bytes(b"hello");
}
```
[Playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=6af0a4ba5de9205cc7ba1a94a623bbfd)

In your example, it'd be `br#"...",`.